### PR TITLE
Added stats component to show likes, comments, and share button

### PR DESCRIPTION
**Pull request by etamity**
For share card and share detail

### CODE REVIEW

File: src/elements/kw-share-stats/demo/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            margin-left: auto;
+            margin-right: auto;
+        }
+        
+    </style>
+</head>
+
+<body>
+    <div class="vertical-section-container centered" style="background-color:#27323A">
+        <h3>Basic kw-share-shell demo</h3>
```
Comment (paulvarache) [inline src/elements/kw-share-stats/demo/index.html]: Copy and paste typo

File: src/elements/kw-share-stats/demo/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            max-width: 620px;
+            margin-left: auto;
+            margin-right: auto;
+        }
+        
+    </style>
+</head>
+
+<body>
+    <div class="vertical-section-container centered" style="background-color:#27323A">
```
Comment (paulvarache) [inline src/elements/kw-share-stats/demo/index.html]: Inline style, no CSP comppliant

File: src/elements/kw-share-stats/demo/index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        <h3>Basic kw-share-shell demo</h3>
+        <demo-snippet>
+            <template>
+               <kw-share-stats id="stats"> </kw-share-stats>
+                <script>
+                        document.addEventListener('WebComponentsReady', function () {
+                            var cardshell = document.querySelector('kw-share-stats#stats');
```
Comment (paulvarache) [inline src/elements/kw-share-stats/demo/index.html]: Old variable name for shareCard, copy and paste typo

File: src/elements/kw-share-stats/kw-share-stats.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    <script>
+        Polymer({
+            is: 'kw-share-stats',
+            properties: {
+                share: {
+                    type: Object
+                }
+            },
+            doLikeAction(){
+                this.fire("likeAction");
```
Comment (paulvarache) [inline src/elements/kw-share-stats/kw-share-stats.html]: We use a dash case styling for events. Check https://www.w3.org/TR/html-markup/syntax.html#syntax-attributes again

File: src/elements/kw-share-stats/kw-share-stats.html [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+            }
+            
+            div.stats[tombstone] .likes span img,
+            div.stats[tombstone] .comments span img,
+            div.stats[tombstone] .shares span img {
+                display: none;
+            }
+        </style>
+        <div class="stats">
+            <button class="likes" on-tap="doLikeAction">
```
Comment (paulvarache) [inline src/elements/kw-share-stats/kw-share-stats.html]: A method name matching the actual event and not the action would be more fitting something like `_onLikeTapped`

File: src/elements/kw-share-stats/kw-share-stats.html [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+            <button class="shares" on-tap="doShareAction">
+                <span><img src="/assets/icons/shares/share.svg"> </img>share</span>
+            </button>
+        </div>
+    </template>
+    <script>
+        Polymer({
+            is: 'kw-share-stats',
+            properties: {
+                share: {
```
Comment (paulvarache) [inline src/elements/kw-share-stats/kw-share-stats.html]: I think that for the size of the component, you could actually just provide the three properties you need and not the whole object. It is more efficient and more reusable.

**Comment by etamity** [comment]
Updated,  I will update all other component which using the stats

File: src/elements/kw-share-stats/demo/index.html [hunk 7]
```diff
@@ -11,31 +11,16 @@
...
-                        document.addEventListener('WebComponentsReady', function () {
-                            var cardshell = document.querySelector('kw-share-stats#stats');
-                             var share = {
-                                likes: new Array(27),
-                                comments_count: 5
-                            }
-                           cardshell.share = share;
-                        });
-                    </script>
+               <kw-share-stats id="stats" likes=100 comments=204> </kw-share-stats>
```
Comment (paulvarache) [inline src/elements/kw-share-stats/demo/index.html]: Attributes are defined between quotes

**Review by paulvarache — COMMENTED** [review]
Looks good to me, just one thing to change in the demo and we're good to go

**Review by paulvarache — APPROVED** [review]
When we're not in freeze anymore, this is good to go

**Pull request merged**